services/ResourceService/composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "islandora/resource-service",
+    "description": "RESTful service providing resources in Fedora 4",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "/Users/dpino/Desktop/Development/ISLANDORAWORK/CLAW_MICRO/chullo"
```

Comment:
Duplicate of below comment: Should this be the main github repo and just make this PR rely on the Chullo PR?